php-checks/src/test/java/org/sonar/php/checks/utils/CheckUtilsTest.java

```diff
@@ -187,6 +193,41 @@ public void trim_quotes_literal_on_non_string() {
     trimQuotes((LiteralTree) expressionFromStatement("42;"));
   }
 
+  @Test
+  public void is_false_value() throws Exception {
+    List<Boolean> falseValues = createLiterals(Tree.Kind.BOOLEAN_LITERAL, "false", "False", "FALSE")
+      .map(CheckUtils::isFalseValue).collect(Collectors.toList());
+    assertThat(falseValues).containsOnly(true);
```

Comment:
This is a bit confusing. What about something like the following?
```
assertThat(createLiterals(Tree.Kind.BOOLEAN_LITERAL, "false", "False", "FALSE").allMatch(CheckUtils::isFalseValue));
```
(I didn't check it...)